packages/component-library-angular/src/table-header-cell/component.ts

```diff
@@ -12,6 +12,6 @@ import { ChangeDetectionStrategy, Component, Input, ViewEncapsulation } from '@a
   },
 })
 export class UtrechtTableHeaderCellAttr {
-  @Input() scope: string = null;
+  @Input() scope?: string;
```

Comment:
I had used `scope = null` instead of `scope?` because in the DOM for Elements usually have properties taht are `null` if they are not set instead of `undefined`, and I meant to mimic that behavior for our custom elements.

Reply:
I changed to `scope?:` because the TypeScript was complaining about  `Type 'null' is not assignable to type 'string'.ts(2322)`  but I can change ito `@Input() scope: string | null` ?